### PR TITLE
2 instances of lists made into lists

### PR DESCRIPTION
**Pull request by morrowc**
Reading a list inline is janky, pulling the list content into an actual list makes it easier to find later and clear that these are N requirements.

### CODE REVIEW

File: standards/SPIFFE.md [hunk 1]
```diff
@@ -27,9 +27,17 @@ While each of these components has a dedicated specification, the remainder of t
...
+A SPIFFE Verifiable Identity Document (SVID) is a document which carries the SPIFFE ID itself. It is the functional equivalent of a passport - a document which is presented that carries the identity of the presenter. Of course, similar to passports, they must be resistant to forgery, and it must be obvious that the document belongs to the presenter. In order to achieve this, an SVID includes cryptographic properties which allow it to be:
 
-An SVID itself is not a document type. Instead, we define 1) the properties required of an SVID, and 2) the method by which SVID information can be encoded and validated in various existing document types. The supported document types are an X.509 certificate or a JWT token.
+  1) proven as authentic
```
Comment (azdagron) [inline standards/SPIFFE.md:32]: Should be consistent with the punctuation at the end of the list item (period or not).

File: standards/SPIFFE.md [hunk 2]
```diff
@@ -27,9 +27,17 @@ While each of these components has a dedicated specification, the remainder of t
...
+  2) proven to belong to the presenter.
+
+An SVID itself is not a document type. Instead, we define:
+
+  1) the properties required of an SVID
```
Comment (azdagron) [inline standards/SPIFFE.md:37]: Same here, should be consistent with the punctuation at the end of the list item (period or not).

**Review by azdagron — COMMENTED** [review]
This looks good to me. Just a few comments.

File: standards/SPIFFE.md (hunk 2 above)
Comment (morrowc) [reply standards/SPIFFE.md:37]: absolutely. done.

File: standards/SPIFFE.md (hunk 1 above)
Comment (morrowc) [reply standards/SPIFFE.md:32]: done.

**Comment by azdagron** [comment]
Thanks for touching that up. One more thing; all commits into the SPIFFE organization require DCO signoff. Please see the failed check for details.

**Comment by morrowc** [comment]
ok, I think some  git history-edit/rewording/etc and DOC passes now, sweet! (and I learned something!)

**Comment by morrowc** [comment]
Do I need to assign this to a code owner?
oh, nm it is assigned ok :)

**Comment by morrowc** [comment]
Chelo..."friendly ping" and all of that stuff.

**Review by azdagron — APPROVED** [review]
Thanks!

**Pull request merged**